CHANGELOG.md

```diff
@@ -65,6 +65,14 @@ Replace any use of this class with the `govuk-tag--grey` class.
 
 This change was introduced in [pull request #2417: Remove deprecated `govuk-tag--inactive class`](https://github.com/alphagov/govuk-frontend/pull/2417).
 
+#### Style the `html` in your cookie banner
```

Comment:
Are we referring here to HTML-the-thing or to the `html` parameter? If it's the thing, then we could just write HTML here.

```suggestion
#### Style the HTML in your cookie banner
```

Reply:
Hm, kind of both 😅 It's if someone uses the `html` parameter to pass HTML code in, rather than using the `text`parameter and just passing in a string of text.

Reply:
Although actually, thinking about it, I think the same applies if a user is using the HTML version of the component rather than the macros. So maybe we need to be more generic and just talk about any custom HTML a user provides... 

I'll have a go at tweaking it and see what I come up with.

Reply:
@EoinShaughnessy I've had a go at rewording this a bit to be more about "custom HTML" and less about how the user is providing that custom HTML. Would you mind taking another look?